src/ui/toolbar/BatteryIndicator.qml

```diff
@@ -205,6 +210,55 @@ Item {
                         }
                     }
                 }
+
+                QGCLabel {
+                    Layout.alignment:   Qt.AlignCenter
+                    text:               qsTr("Limits")
+                    font.family:        ScreenTools.demiboldFontFamily
+                }
+
+                RowLayout {
+                    spacing: ScreenTools.defaultFontPixelWidth
+
+                    ColumnLayout {
+                        QGCLabel { text: _aviantSettings.preDeliveryBingoLimit.name }
+                        QGCLabel { text: _aviantSettings.postDeliveryBingoLimit.name }
+                        QGCLabel { 
+                            text: "Battery Failsafe"
+                            color:  qgcPal.colorOrange
+                        }
+                        QGCLabel {
+                            text: "Battery Emergency"
+                            color:  qgcPal.colorRed
+                        }
+                        QGCLabel {
+                            text: "Battery capacity"
+                        }
+                    }
+                    ColumnLayout {
+                        id: limitValues
+
+                        QGCLabel { text: _aviantSettings.preDeliveryBingoLimit.rawValue + " " + _aviantSettings.preDeliveryBingoLimit.units }
+                        QGCLabel { text: _aviantSettings.postDeliveryBingoLimit.rawValue + " " + _aviantSettings.postDeliveryBingoLimit.units }
+                        QGCLabel {
+                            text:  limitValues.thresholdPercentageToMah(controller.getParameterFact(-1, "BAT_CRIT_THR").rawValue, 16000) + " " + controller.getParameterFact(-1, "BAT1_CAPACITY").units
+                            color: qgcPal.colorOrange
+                        }
+                        QGCLabel { 
+                            text:  limitValues.thresholdPercentageToMah(controller.getParameterFact(-1, "BAT_EMERGEN_THR").rawValue, 16000) + " " +controller.getParameterFact(-1, "BAT1_CAPACITY").units
```

Comment:
it feels a little too detailed to call direct parameter names here, is that needed or is there a nicer solution?

Reply:
I dont think there is a nicer solution, it is eg. how it is done [here](https://github.com/aviant-tech/qgroundcontrol/blob/feat/battery-widget/src/AutoPilotPlugins/PX4/SafetyComponent.qml#L130)

Reply:
the referred page is only loaded in "Safety settings" which comes after the firmware plugin is loaded

Reply:
Pushed a commit fixing the problem